webpack/ForemanTasks/Components/TasksTable/TasksTableHelpers.js

```diff
@@ -1,4 +1,8 @@
 import URI from 'urijs';
+import { translate as __, documentLocale } from 'foremanReact/common/I18n';
+import humanizeDuration from 'humanize-duration';
```

Comment:
Foreman core uses React-intl and while they doesn't support duration API yet,
there is a lib called [react-intl-formatted-duration](https://github.com/en-japan-air/react-intl-formatted-duration) which support the React-intl provider so you don't need to configure all the localizations by yourself.

Reply:
it looks like this library is less used, and updated less often, I'm not sure we should use it

Reply:
Does it supports `i18n` like `react-intl`?
I didn't test the `react-intl-formatted-duration` `i18n` but I think it worth to investigate it.
@MariaAga could you add it into our backlog? thanks

Reply:
Added to the backlog, thanks